### PR TITLE
[ISSUE #4011] add kafkaConnector module

### LINKED ISSUE #4011: [Feature] Add Source Connector Kafka, Sink Connector Kafka module
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Feature Request

Task list:

 - add sink-connector-kafka module
 - add source-connector-kafka module

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by epiao55**
Fixes #4011.
add kafkaConnector module, include sink and source

### CODE REVIEW

**Comment by epiao55** [comment]
@qqeasonchen @xwm1992  PTAL

**Review by mxsm — COMMENTED** [review]
@epiao55  Please fix code style

**Comment by epiao55** [comment]
I have fixed style @mxsm

**Comment by epiao55** [comment]
PTAL~ @xwm1992

**Review by xwm1992 — APPROVED** [review]

**Review by Alonexc — APPROVED** [review]

File: eventmesh-connectors/eventmesh-connector-kafka/src/main/java/org/apache/eventmesh/connector/kafka/source/connector/KafkaSourceConnector.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public void init(Config config) throws Exception {
+        this.sourceConfig = (KafkaSourceConfig) config;
+        Properties props = new Properties();
+        props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, sourceConfig.getConnectorConfig().getBootstrapServers());
+        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        props.put(ConsumerConfig.GROUP_ID_CONFIG, sourceConfig.getConnectorConfig().getGroupID());
+        props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
+
+        this.kafkaConsumer = new KafkaConsumer<String, String>(props);
```
Comment (mxsm) [inline eventmesh-connectors/eventmesh-connector-kafka/src/main/java/org/apache/eventmesh/connector/kafka/source/connector/KafkaSourceConnector.java:62]: How about add max.poll.records to KafkaConsumer?The exact numbers are placed in the configuration file.

Comment (epiao55) [reply eventmesh-connectors/eventmesh-connector-kafka/src/main/java/org/apache/eventmesh/connector/kafka/source/connector/KafkaSourceConnector.java]: ok，i have added some defalut config for kafka connector

**Comment by epiao55** [comment]
@xwm1992 please help trigger the build, tks

**Review by mxsm — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**